src/python/pants/backend/python/util_rules/pex_environment.py

```diff
@@ -105,11 +105,12 @@ class PexEnvironment(EngineAwareReturnType):
     bootstrap_python: Optional[PythonExecutable] = None
 
     def create_argv(
-        self, pex_path: str, *args: str, python: Optional[PythonExecutable] = None
+        self, pex_filepath: str, *args: str, python: Optional[PythonExecutable] = None
     ) -> Tuple[str, ...]:
         python = python or self.bootstrap_python
         argv = [python.path] if python else []
-        argv.extend((pex_path, *args))
+        argv.append(pex_filepath)
+        argv.extend(args)
```

Comment:
Same behavior, but I think simpler. I suspect performance is a wash; we had to create a tuple before, then throw it away.